identity/src/main/java/com/stripe/android/identity/viewmodel/CameraViewModel.kt

```diff
@@ -66,6 +73,9 @@ internal open class CameraViewModel(
 
     override fun displayState(newState: IdentityScanState, previousState: IdentityScanState?) {
         displayStateChanged.postValue(newState to previousState)
+        displayStateChangedFlow.update {
+            (newState to previousState)
```

Comment:
This functionality seems useful, maybe we can extract it to an extension function? something like:

```
fun <T: Any> Flow<T>.withPrevious(): Flow<Pair<T?, T>> = flow {
    var prev: T? = null
    this@withPrevious.collect {
        emit(prev to it)
        prev = it
    }
}
```

Reply:
hmm just tried this and seems it's not exactly same as we needed here as `displayStateChangedFlow` could have a `null` pair value, so we would need to write it as follows, not sure how reusable this would be.. so would keep it as is, wdyt?
```
fun <T: Any> Flow<T?>.withPrevious(): Flow<Pair<T?, T>?> = flow {
    var prev: T? = null

    this@withPrevious.collect {
        it?.let {
            emit(prev to it)
            prev = it
        }
        emit(null)
    }
}
```

(Note the reason we need to start with null is because we can't just assign a initial `IdentityScanState` right away, it is decided by subclass)